tf2onnx/custom_opsets/ms.py

```diff
@@ -103,3 +103,11 @@ def version_11(cls, ctx, node, **kwargs):
         ctx.insert_new_node_on_input(node, "Transpose", node.input[0], perm=constants.NHWC_TO_NCHW)
         ctx.insert_new_node_on_output("Transpose", node.output[0], node.name + '_transposed',
                                       None, perm=constants.NCHW_TO_NHWC)
+
+@tf_op("MatrixInverse", onnx_op="Inverse")
```

Comment:
domain= is missing.

Reply:
And since you just moved it, it doesn't satisfy the onnx spec so it does not work for trt as example. To implement the onnx spec you need to keep it in math.py as well.

Reply:
Done.